sdks/python/apache_beam/runners/worker/opcounters.py

```diff
@@ -42,6 +43,58 @@ def value(self):
     return self._value
 
 
+class TransformIoCounter(object):
+
+  def add_bytes_read(self, n):
+    pass
+
+  def __enter__(self):
+    self.enter()
+
+  def __exit__(self, unused_exc_type, unused_exc_value, unused_traceback):
+    self.exit()
+
+  def enter(self):
+    pass
+
+  def exit(self):
+    pass
+
+  def check_step(self):
+    pass
+
+
+class SideInputReadCounter(TransformIoCounter):
+
+  def __init__(self, counter_factory, state_sampler, io_target):
+    self._counter_factory = counter_factory
+    self._state_sampler = state_sampler
+    self._bytes_read_cache = 0
+    self.io_target = io_target
+    self.check_step()
```

Comment:
What does this do? Documentation on the parent class would be helpful even if it's not user-facing. The implementation below doesn't look like it's checking stuff.